### ISSUE TITLE
Property test failure for `attoparsec-0.13.1.0`

### ISSUE THREAD

**Issue by Gabriel439** [bos/attoparsec#134]
```
    satisfyWith:             FAIL
      *** Failed! Falsifiable (after 2 tests): 
      '\961347'
      ""
      Just 'C' /= Just '\961347'
      Use --quickcheck-replay '1 TFGenR 000000074CAFD361000000000007A120000000000000E2980000000A41C36F40 0 384 9 0' to reproduce.
```

**Comment by hvr** [comment]
@Gabriel439 this was fixed in attoparsec-0.13.2.0

**Comment by basvandijk** [comment]
I get the same failure with `attoparsec-0.13.2.0` and `QuickCheck-2.10.1` (I jailbreaked attoparsec):

```
    satisfyWith:             FAIL
      *** Failed! Falsifiable (after 2 tests): 
      '\754924'
      ""
      Just '\236' /= Just '\754924'
      Use --quickcheck-replay '1 TFGenR 00002F4BE1597EF1000000003B9ACA00000000000000E3080000000BDBE2A158 0 384 9 0' to reproduce.
```

**Comment by hvr** [comment]
Well, yes; if you override the version constraints it's not surprising the test-suite may misbehave... there was a reason for that upper bound on QC... the test-suite simply isn't yet QC-2.10 compatible (and just to give all clear; this does *not* point to a bug in attoparsec, but rather to the test-case relying on the semantics of `Arbitrary Char` which got changed in QC 2.10, and this semantic change one of the primary reasons that QC did a major ver bump.)

**Comment by Gabriel439** [comment]
@hvr: Sorry, I missed your original comment.  I'll mark this closed since it's fixed in `attoparsec-0.13.2.0`
